addons/jest/src/components/Panel.tsx

```diff
@@ -117,6 +123,7 @@ const Content = styled(({ tests, className }: ContentProps) => (
       }
 
       const testsByType: Map<string, any> = getTestsByTypeMap(result);
+      console.log({ testsByType });
```

Comment:
Since I dont have access to change, Ill remove this after its merged to my branch